### DISCUSSION TITLE
[Feature] 'Upload to Immich' from OS share interface

### DISCUSSION THREAD

**Discussion by ResponsiblyUnranked** [immich-app/immich#8534] [Feature Request]
### The feature

When I 'share' a photo on my Android phone I can send it to a multitude of apps. One of these is an 'Upload to app-which-must-not-be-named' option.

I find this useful, for example, because I don't want to back-up all photos from the 'Whatsapp' folder on my phone, but it would be nice to be able to upload a few specific ones.

This is the button I'm talking about (middle option):
![signal-2024-04-05-07-21-26-916](https://github.com/immich-app/immich/assets/51296820/0e6ea96a-c439-427d-841f-0dc49011ffdc)

It would also allow me to upload images to Immich from many different sources, rather than the standard system folder backup function.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2659

**Reply by ResponsiblyUnranked** [reply]
Ahh thanks! Tried to search but couldn't find this one.
